mmv1/products/redis/api.yaml

```diff
@@ -190,3 +190,43 @@ objects:
           - :STANDARD_HA
         default_value: :BASIC
         input: true
+      - !ruby/object:Api::Type::Enum
+        name: transitEncryptionMode
+        min_version: beta
+        input: true
+        description: |
+          The TLS mode of the Redis instance, If not provided, TLS is disabled for the instance.
+
+          - SERVER_AUTHENTICATION: Client to Server traffic encryption enabled with server authentcation
+        values:
+          - :SERVER_AUTHENTICATION
+          - :DISABLED
+        default_value: :DISABLED
+      - !ruby/object:Api::Type::Array
+        name: 'serverCaCerts'
+        min_version: beta
+        description: |
+          List of server CA certificates for the instance.
+        output: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
```

Comment:
Can you mark each subproperty here `output: true` as well? That'll make this schema a little more correct: [`auto-pr-4444-old..auto-pr-4444`#diff-fd32ba4f30](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-4444-old..auto-pr-4444#diff-fd32ba4f30d5fbeda7a41643508b29476cec3dae90ad833de3d062bd2cf20fceR210-R243)